### ISSUE TITLE
[logback] Class path contains multiple SLF4J bindings

### ISSUE THREAD

**Issue by diogosantana** [psi-probe/psi-probe#620]
After #611 logback shows the message `Class path contains multiple SLF4J bindings` because of openejb dependency slf4j-jdk14-1.7.7.jar is on the test class path.

I can only check whether this affects test cases, but I can not test TomcatEE installations, which maybe be affected by this conflict. I will open another issue to remember to test it.

**Comment by hazendaz** [comment]
I've got a local change to cleanup dependencies we use during the build which will exclude all of openEJB core dependencies since we don't need them on the build.  That will fix the build warning.

TomEE is obvious problem :(  I've got this same issue open on Waffle project for long time.  No one ever actually complained.  So possible solution here can fix both libraries in a positive way :)

**Comment by hazendaz** [comment]
TomEE from a old nabble issue I read seems like this would be done all it to load the WAR version.  I think this gets added in our WAR.

context.xml
<?xml version="1.0" encoding="UTF-8"?>
<Context >
    <Loader delegate="false" />
</Context>
